### PR TITLE
feat(Footer): add footer components

### PR DESCRIPTION
**Pull request by kseniya57**
The page footer components

### CODE REVIEW

File: src/components/Footer/desktop/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            min-width: 0;
+
+            visibility: hidden;
+
+            &_measured {
+                visibility: visible;
+            }
+        }
+
+        &__list[class] {
```
Comment (Lunory) [inline src/components/Footer/desktop/Footer.scss:30]: `&__list`?

Comment (kseniya57) [reply src/components/Footer/desktop/Footer.scss:30]: Это для переопределения стандартных стилей, там приоритет больше

File: src/components/Footer/desktop/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            background-color: inherit;
+            display: flex;
+
+            height: 100%;
+            padding: 0;
+
+            .g-menu__list-item {
+                height: 100%;
+                padding: 0;
+
```
[the remark this answers is not in the document]
Comment (kseniya57) [reply src/components/Footer/desktop/Footer.scss:40]: We can't do that for the menu item for now, as it doesn't provide the property for the container class

Screenshot 2024-03-07 at 08 24 33

[the remark this answers is not in the document]
Comment (kseniya57) [reply src/components/Footer/desktop/Footer.scss]: @Lunory Thanks, I've fixed this, seems like the menu was updated since in the new uikit)

**Review by Lunory — APPROVED** [review]

**Pull request merged**